src/components/themed/ControlPanel.js

```diff
@@ -104,8 +101,21 @@ export function ControlPanel(props: Props) {
   }
 
   const handleSweep = () => {
+    const sweepableCurrencyCodes = Object.keys(SPECIAL_CURRENCY_INFO)
```

Comment:
why did `sweepableCurrencyCodes` get moved? now the function would need to get defined on every call to `handleSweep`. Move back to original location